### PR TITLE
GamaPlatform.Gama version 1.9.1

### PR DESCRIPTION
**Pull request by lesquoyb**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/105878)

### CODE REVIEW

**Comment by lesquoyb** [comment]
@microsoft-github-policy-service agree company="Gama-platform"

**Comment by stephengillie** [comment]
Automatic Validation ended with:
InstallationVerification-x64-Scope_Undefined-Locale_Undefined-ErrorScreenshot_20230504102828

File: manifests/g/GamaPlatform/Gama/1.9.1/GamaPlatform.Gama.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+PackageIdentifier: GamaPlatform.Gama
+PackageVersion: 1.9.1
+Installers:
+- Architecture: x64
+  InstallerType: inno
+  InstallerUrl: https://github.com/gama-platform/gama/releases/download/1.9.1/GAMA_1.9.1_Windows_with_JDK.exe
+  InstallerSha256: 038130F03824EAFB84070D168906B3B124406D607D5EC64A9941190979E69D18
+  FileExtensions:
+  - gaml
+  ReleaseDate: 04-27-2023
```
Comment (stephengillie) [inline manifests/g/GamaPlatform/Gama/1.9.1/GamaPlatform.Gama.installer.yaml:13]:
```suggestion
InstallerType: inno
Installers:
- Architecture: x64
  Scope: user
  InstallerUrl: https://github.com/gama-platform/gama/releases/download/1.9.1/GAMA_1.9.1_Windows_with_JDK.exe
  InstallerSha256: 038130F03824EAFB84070D168906B3B124406D607D5EC64A9941190979E69D18
  InstallerSwitches:
    Custom: /CurrentUser
- Architecture: x64
  Scope: machine
  InstallerUrl: https://github.com/gama-platform/gama/releases/download/1.9.1/GAMA_1.9.1_Windows_with_JDK.exe
  InstallerSha256: 038130F03824EAFB84070D168906B3B124406D607D5EC64A9941190979E69D18
  InstallerSwitches:
    Custom: /AllUsers
FileExtensions:
- gaml
ReleaseDate: 04-27-2023
```

**Comment by stephengillie** [comment]
[Policy 2.7](https://learn.microsoft.com/en-us/windows/package-manager/package/windows-package-manager-policies#27-adult-content)

**Review by stephengillie — APPROVED** [review]

**Pull request merged**